aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -133,6 +133,56 @@ const char* _cublasGetErrorEnum(cublasStatus_t error) {
 
 /* LEVEL 3 BLAS FUNCTIONS */
 
+#ifndef __HIP_PLATFORM_HCC__
+#if defined(CUDA_VERSION) && CUDA_VERSION >= 11200
```

Comment:
Is this macro `CUDA_VERSION >= 11200` intended? If you mean cuda 11.2, it should be 11020. I'm not sure if cuda 11.2 was a thing back in November 2020. 😅

Reply:
No harm done, workaround is good.

Reply:
@xwang233 no my bad! we should fix that to avoid confusion in future